{"uuid": "a08d7a05-b1fd-4c00-b5bd-7297aba60665", "children": ["33f11069-9710-4296-b590-846669c9507c"], "befores": [{"name": "driver", "status": "passed", "start": 1691765132964, "stop": 1691765134567}], "afters": [{"name": "driver::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 917, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\conftest.py\", line 15, in driver\n    if request.node.rep_call.failed:\n       ^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1691765136882, "stop": 1691765136883}], "start": 1691765132964, "stop": 1691765136883}